changelog.json

```
@@ -1,5 +1,21 @@
 {
   "releases": {
+    "2.4.1-beta1": [
+      "[Added] Indicate which commits are un-pushed in \"History\" tab - #5873",
+      "[Added] Show progress bar in taskbar during pushing and pulling changes - #8433. Thanks @ChamodyaDias!",
+      "[Added] Add support for PyCharm as external editor on macOS - #9401. Thanks @jakubroztocil!",
+      "[Added] Add support for Notepad++ - #9235. Thanks @say25!",
+      "[Added] Add support for Android Studio on macOS - #9332. Thanks @guidezpl!",
+      "[Added] Add support to Rider on Mac - #9368. Thanks @wengfatt!",
+      "[Added] Add support for JetBrains Rider on Windows - #9365. Thanks @say25!",
```

Reordered a bit the external editor changes and joined both JetBrains Rider entries:

```suggestion
      "[Added] Add support for JetBrains Rider as external editor - #9368 #9365. Thanks @wengfatt and @say25!",
      "[Added] Add support for PyCharm as external editor on macOS - #9401. Thanks @jakubroztocil!",
      "[Added] Add support for Android Studio as external editor on macOS - #9332. Thanks @guidezpl!",
      "[Added] Add support for Notepad++ as external editor on Windows - #9235. Thanks @say25!",
```